python_on_whales/components/image/cli_wrapper.py

```diff
@@ -476,7 +476,7 @@ def prune(self, all: bool = False, filter: Dict[str, str] = {}) -> str:
         return run(full_cmd)
 
     def pull(
-        self, x: Union[str, List[str]], quiet: bool = False
+        self, x: Union[str, List[str]], quiet: bool = False, platform: str = ""
```

Comment:
Would be better to make this `platform: Optional[str] = None` to be clear about the intent of not specifying a platform (rather than specifying an empty string potentially having meaning).